### ISSUE TITLE
[Bug] Tab stops not showing focused elements

### ISSUE THREAD

**Issue by ThanyaLeif** [microsoft/accessibility-insights-web#6042]
## Describe the bug

Accessibility Insight Tab stop is not working on [MSN](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.msn.com%2F&data=05%7C01%7Ctaniamar%40microsoft.com%7C1843272f1a65466a3bb608da968560a3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637987797091396725%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sF3XGt8O74uewRBI5SNdIHD%2FYXgDn3mb3RkK7ywIrNs%3D&reserved=0). Focused elements should be highlighted, but only a single tab in the middle of the page is marked.

**To Reproduce**

1. Go to [MSN portal](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.msn.com%2F&data=05%7C01%7Ctaniamar%40microsoft.com%7C1843272f1a65466a3bb608da968560a3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637987797091396725%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sF3XGt8O74uewRBI5SNdIHD%2FYXgDn3mb3RkK7ywIrNs%3D&reserved=0)
2. Go to Accessibility Extension > Ad hoc tools
3. Turn on Tab stops
4. Navigate throw the page using `Tab` key
5. Notice how the focused elements are not marked as tab stops.

## Expected behavior

All focused elements should be marked as Tab stops by the extension.

## Screenshots

![image](https://user-images.githubusercontent.com/32908302/190509287-e3f0f8f5-69ae-405d-8186-4b71c7f2b93f.png)
Image of a single tab stop shown after Tab stop test.

## Context (please complete the following information)

- AI-Web Version & Environment: Production Version 2.34.1
- Browser Version: Version 105.0.1343.33 (Official build) (64-bit)
- Target Page: https://www.msn.com/

## Are you willing to submit a PR?
Yes

## Did you search for similar existing issues?
Yes

**Comment by ThanyaLeif** [comment]
After further investigation, we believe that the cause of the problem is msn.com's implementation of fluent user interface web components. Most of the buttons are located inside closed shadow doms. Accessibility Insighst can only scan open shadow doms, here is a link with more information about it: 
[https://accessibilityinsights.io/docs/web/reference/faq/#are-automated-checks-compatible-with-shadow-dom](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2Fdocs%2Fweb%2Freference%2Ffaq%2F%23are-automated-checks-compatible-with-shadow-dom&data=05%7C01%7Ctaniamar%40microsoft.com%7Cbc1a3615eb044060c7f808da98399bbe%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637989670518743732%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=4pFUYBWHetUof7ALncRYMm0YQV2XyrAneg0adClFc9M%3D&reserved=0)
